--- a/frontend/lib/database_api/frb_generated.dart.unpatched
+++ b/frontend/lib/database_api/frb_generated.dart
@@ -73,7 +73,7 @@ class RustLib extends BaseEntrypoint<RustLibApi, RustLibApiImpl, RustLibWire> {

   static const kDefaultExternalLibraryLoaderConfig =
       ExternalLibraryLoaderConfig(
-    stem: 'UNKNOWN',
+    stem: 'backend_api',
     ioDirectory: '../backend/backend_api/target/release/',
     webPrefix: 'pkg/',
   );
