--- a/backend/interface/database_api/src/api/models.unpatched.rs
+++ b/backend/interface/database_api/src/api/models.rs
@@ -17,6 +17,7 @@ use crate::schema::*;
 #[derive(Queryable,QueryableByName,Selectable,Serialize,Deserialize,JsonFieldConversionGenerator, Identifiable)]
 #[diesel(primary_key(membership_id))]
 #[diesel(table_name = member)]
+#[make_fields_non_final]
 pub struct Member {
     pub membership_id: i32,
     pub prename: String,
