--- a/frontend/lib/interface/database_api/frb_generated.dart.unpatched
+++ b/frontend/lib/interface/database_api/frb_generated.dart
@@ -71,7 +71,7 @@ class RustLib extends BaseEntrypoint<RustLibApi, RustLibApiImpl, RustLibWire> {

   static const kDefaultExternalLibraryLoaderConfig =
       ExternalLibraryLoaderConfig(
-        stem: 'UNKNOWN',
+        stem: 'database_api',
         ioDirectory: '../backend/database_api/target/release/',
         webPrefix: 'pkg/',
       );
