--- a/lib/backend/models.dart.unpatched
+++ b/lib/backend/models.dart
@@ -5,9 +5,11 @@

 import 'frb_generated.dart';
 import 'package:flutter_rust_bridge/flutter_rust_bridge_for_generated.dart';
+import 'package:green3neo/reflectable.dart';

 // Rust type: RustOpaqueMoi<flutter_rust_bridge::for_generated::rust_async::RwLock<Member>>
 @sealed
+@reflectableMarker
 class Member extends RustOpaque {
   Member.dcoDecode(List<dynamic> wire) : super.dcoDecode(wire, _kStaticData);

